evaluation/apps.py

```diff
@@ -0,0 +1,10 @@
+"""
+Project OCELoT: Open, Competitive Evaluation Leaderboard of Translations
+"""
+from django.apps import AppConfig
+
+
+class EvaluationConfig(AppConfig):
+    """ocelot evaluation app Configuration"""
```

Comment:
OCELoT

Reply:
`OCELoT evaluation app configuration`

Reply:
Fixed in both `leaderboard/apps.py` and `evaluation/apps.py`